system/lib/wasmfs/wasmfs.h

```diff
@@ -41,7 +41,8 @@ class WasmFS {
   // Files will be preloaded in this constructor.
   // This global constructor has init_priority 100. Please see wasmfs.cpp.
   // The current working directory is initialized to the root directory.
-  WasmFS() : rootDirectory(initRootDirectory()), cwd(rootDirectory) {
+  WasmFS()
+    : backendTable(), rootDirectory(initRootDirectory()), cwd(rootDirectory) {
```

Comment:
This change isn't actually necessary because `backendTable` will already be default-initialized implicitly.